src/mgr/BaseMgrModule.cc

```diff
@@ -1491,7 +1528,10 @@ PyMethodDef BaseMgrModule_methods[] = {
     "Get a performance counter"},
 
   {"_ceph_get_latest_unlabeled_counter", (PyCFunction)get_latest_unlabeled_counter, METH_VARARGS,
-    "Get the latest performance counter"},
+    "Get the latest value for unlabeled performance counter"},
```

Comment:
Not sure about the phrasing here (and 3 lines below).

Reply:
I have updated the phrasing 3 lines [below](https://github.com/ceph/ceph/pull/61997/files#diff-418325979416216a8730e1e2bb75ab0f24c998ca6e8f4899162125b39fe315c4R1537), but can you please explain why do you think the phrasing is incorrect on this line ^^'

Reply:
Fetch (or get) the latest (or updated) value of an unlabeled counter

Reply:
done!